src/app/core/services/label.service.ts

```diff
@@ -6,6 +6,15 @@ import { Observable } from 'rxjs';
 import { SEVERITY_ORDER } from '../../core/models/issue.model';
 import { User } from '../models/user.model';
 
+/* The threshold to decide if color is dark or light.
+A higher threshold value will result in more colors determined to be "dark".
+W3C recommendation is 0.179, but 0.184 is chosen so that some colors (like bright red)
+have white text instead of black. Github labels are also red/white instead of red/black */
+const COLOR_DARKNESS_THRESHOLD = 0.184;
+
+const LIGHT_COLOR = 'FFFFFF'; // Light color for text with dark background
```

Comment:
`COLOR_LIGHT_TEXT` `COLOR_DARK_TEXT` ?

Reply:
Reason: prefixing related constants the same way will make them appear together when sorted alphabetically.